mod/admin.php

```diff
@@ -868,6 +869,14 @@ function admin_page_summary(App $a)
 		$warningtext[] = L10n::t('Friendica\'s configuration now is stored in config/local.ini.php, please copy config/local-sample.ini.php and move your config from <code>.htconfig.php</code>. See <a href="%s">the Config help page</a> for help with the transition.', $a->get_baseurl() . '/help/Config');
 	}
 
+	// Check server vitality
+	if (!admin_page_server_vital()) {
+		$showwarning = true;
+		$well_known = $a->get_baseurl() . '/.well-known/host-meta';
+		$warningtext[] = L10n::t('<a href="%s">%s</a> is not reachable on your system. This is a servere configuration issue that prevents the communication. See <a href="%s">the installation page</a> for help.',
```

Comment:
"prevents the communication" should be "prevents communication"

Reply:
@miqrogroove nitpicker 😛